CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v2.2.2 July 18, 2016
```

Comment:
@LalatenduMohanty No. Just that I noticed it and found there should be unifying policy across ADB ecosystem projects. :smile: